CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Expand print link component ([PR #2900](https://github.com/alphagov/govuk_publishing_components/pull/2900))
 * Add link click tracking ([PR #2904](https://github.com/alphagov/govuk_publishing_components/pull/2904))
 * Ensure tab clicks grab the tabs href for gtm ([PR #2884](https://github.com/alphagov/govuk_publishing_components/pull/2884))
+* **BREAKING:** Update gtm naming conventions ([PR #2906](https://github.com/alphagov/govuk_publishing_components/pull/2906))
```

Comment:
You're right, this is technically a breaking change, but as this code isn't live in production I'm not sure it warrants a breaking release - I don't want to unnecessarily alarm anyone or make them look through code to check that merging this new release into an app isn't going to break anything. What do you think?

Reply:
Cool makes sense, I'll update the log and merge. Thanks Andy 👍